.github/workflows/npm-publish.yml

```diff
@@ -1,30 +1,26 @@
 name: npm-publish
 on:
   push:
-    branches:
-      - master
+    tags:
```

Comment:
should we have any branch check here?

will it release without merging to master?

@shidil need your input here

Reply:
Only when tags are created. It cannot be created by anyone, so fine

Reply:
> will it release without merging to master?

That is possible

Reply:
Added a few conditions which will make actions to be run when changes are merged to master and with a commit message `release`.
I still doubt `squash` will make this release disappear.

Reply:
Tbh, I like the tag release better. This is fine as well